src/test/scala/cromwell/CromwellTestkitSpec.scala

```diff
@@ -2,14 +2,31 @@ package cromwell
 
 import akka.actor.ActorSystem
 import akka.testkit.{DefaultTimeout, EventFilter, ImplicitSender, TestKit}
+import com.typesafe.config.ConfigFactory
 import org.scalatest.concurrent.ScalaFutures
 import org.scalatest.{BeforeAndAfterAll, Matchers, WordSpecLike}
 
-abstract class CromwellTestkitSpec(actorSystem: ActorSystem) extends TestKit(actorSystem) with DefaultTimeout
-with ImplicitSender with WordSpecLike with Matchers with BeforeAndAfterAll with ScalaFutures {
+object CromwellTestkitSpec {
+  val akkaConfigString =
+    """
+      |akka {
+      |  loggers = ["akka.event.slf4j.Slf4jLogger", "akka.testkit.TestEventListener"]
+      |  loglevel = "DEBUG"
+      |  actor {
+      |    debug {
+      |       receive = on
+      |    }
+      |  }
+      |}
+    """.stripMargin
+}
+
+abstract class CromwellTestkitSpec(name: String)
+  extends TestKit(ActorSystem(name, ConfigFactory.parseString(CromwellTestkitSpec.akkaConfigString)))
+  with DefaultTimeout with ImplicitSender with WordSpecLike with Matchers with BeforeAndAfterAll with ScalaFutures {
 
   def startingCallsFilter(callNames: String*): EventFilter =
-    EventFilter.info(message = s"Starting calls: ${callNames.mkString(", ")}", occurrences = 1)
+    EventFilter.info(pattern = s"starting ${callNames.mkString(", ")}", occurrences = 1)
```

Comment:
oh looks like this isn't too recently rebased, please ignore comments about `waitForPattern` until that happens.